shared/src/commonMain/kotlin/com/innova/innovaPoolingLibrary/DateTimeFormatter.kt

```diff
@@ -0,0 +1,13 @@
+package com.innova.innovaPoolingLibrary
+
+val dateUtilities = DateUtilities()
```

Comment:
all the variables inside DateUtilities has to be static and you it is not recommended to create a anonymous functions